lib/Fetcher/FeedFetcher.php

```diff
@@ -73,7 +73,7 @@ public function fetch(string $url, bool $favicon, $lastModified, $user, $passwor
         }
         $url = $url2->getNormalizedURL();
         $this->reader->resetFilters();
-        if (is_null($lastModified) || !is_string($lastModified)) {
+        if (is_null($lastModified) || !is_string($lastModified) || $lastModified === '0') {
```

Comment:
Might be better to change the `is_null` test to an empty check. Also, this needs tests